client/src/pages/Discover/Discover.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import designerBanner from '../Discover/images/tatto-designer-banner.jpg';
+import blackPattern from '../Discover/images/black-pattern.jpg';
+import useStyles from './useStyles';
+import Navbar from '../../components/Navbar/Navbar';
+import { Contest } from '../../interface/Contest';
+import { useContests } from '../../context/useContestContext';
+import ContestCard from './ContestCard/ContestCard';
+import WinnerCard from './WinnersCard/Winner';
+import winnersTestData from './test data/winnersTestData.js';
+import { Typography, Grid, CssBaseline, Button } from '@material-ui/core';
+import FacebookIcon from '@material-ui/icons/Facebook';
+import InstagramIcon from '@material-ui/icons/Instagram';
+import PinterestIcon from '@material-ui/icons/Pinterest';
+
+export default function Discover(): JSX.Element {
+  const classes = useStyles();
+
+  const [activeContests, setActiveContests] = useState<Contest[] | []>([]);
+  const { allContests } = useContests();
+  const history = useHistory();
+
+  useEffect(() => {
+    const newActiveContests = allContests.filter((contest: Contest) => new Date(contest.deadlineDate) >= new Date());
+    setActiveContests(newActiveContests);
+  }, [allContests]);
+
+  return (
+    <Grid container className={classes.root} direction="column" alignItems="center">
+      <CssBaseline />
+      <Grid item container>
+        <Navbar />
```

Comment:
Also, discuss with the team and maybe render the Navbar just once as opposed to rendering it within every component :)

Reply:
I will communicate this with Daniel and tackle the Navbar issue and routing in a separate pull request. Thank you.